src/xtopcom/xblockstore/src/xvunithub.cpp

```diff
@@ -664,14 +664,21 @@ namespace top
         bool      xvblockstore_impl::exist_genesis_block(const base::xvaccount_t & account) {
             LOAD_BLOCKACCOUNT_PLUGIN(account_obj,account);
```

Comment:
cannot be a `const` member function?
due to `LOAD_BLOCKACCOUNT_PLUGIN`?

Reply:
> 
> 
> cannot be a `const` member function?
> due to `LOAD_BLOCKACCOUNT_PLUGIN`?

yes, it calls a lot of mutable functions in LOAD_BLOCKACCOUNT_PLUGIN